linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/AccessLogger.scala

```diff
@@ -14,13 +14,14 @@ case class AccessLogger(log: Logger) extends SimpleFilter[Request, Response] {
     val user = "-"
     val referer = reqHeaders.getOrElse("Referer", "-")
     val userAgent = reqHeaders.getOrElse("User-Agent", "-")
+    var hostHeader = reqHeaders.getOrElse("Host", "-")
     val reqResource = s"${req.method.toString.toUpperCase} ${req.uri} ${req.version}"
 
     svc(req).onSuccess { rsp =>
       val statusCode = rsp.statusCode
       val responseBytes = rsp.contentLength.map(_.toString).getOrElse("-")
       val requestEndTime = new TimeFormat("dd/MM/yyyy:HH:mm:ss Z").format(Time.now)
-      log.info(s"""$remoteHost $identd $user [$requestEndTime] "$reqResource" $statusCode $responseBytes "$referer" "$userAgent"""")
```

Comment:
We're currently logging in the [Combined Log Format](http://httpd.apache.org/docs/2.4/logs.html#combined), which extends the [Common Log Format](http://httpd.apache.org/docs/2.4/logs.html#common) by adding the referer and user-agent headers at the end. I *think* it would be better to add the host header at the end of the log string rather than at the beginning, as that would allow many tools that interact with the Common or Combined formats to continue working (assuming they ignore any additional fields at the end of the string, etc.). Thoughts?